bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -113,16 +114,35 @@ def test_import_export_workflow_from_local_path(self):
             shutil.rmtree(export_dir)
         self.assertIsInstance(exported_wf_dict, dict)
 
+    def test_import_publish_export_workflow_from_local_path(self):
+        with self.assertRaises(Exception):
+            self.gi.workflows.import_workflow_from_local_path(None)
```

Comment:
I'd remove the previous 2 lines, this is already tested in `test_import_export_workflow_from_local_path()` above.

Reply:
Ok! Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/4b85c7a210e73f22ec263e5655bb395f27ba8157